CRP.Mvc/Services/SlothService.cs

```diff
@@ -68,8 +69,20 @@ public async Task<CreateSlothTransactionResponse> CreateTransaction(CreateTransa
                 var url = "transactions";
 
                 var response = await client.PostAsJsonAsync(url, transaction);
-                var result = await response.GetContentOrNullAsync<CreateSlothTransactionResponse>();
-                return result;
+                try
+                {
+                    var result = await response.GetContentOrNullAsync<CreateSlothTransactionResponse>();
+                    return result;
+                }
+                catch (Exception e)
+                {
+                    Log.Error($"Exception trying to create transaction in sloth: {e.Message}");
```

Comment:
if you are logging the error you should pass the whole error object, not just the exception message